### DISCUSSION TITLE
Add class to focused image afterclose

### DISCUSSION THREAD

**Discussion by ieraora** [fancyapps/ui#547] [Q&A]
Hi. In version 3.5 I've this code to, after close, focused last image viewed in FB and move to him.

afterClose: function(instance, current) {
				$heightwin = $(window).height()/2;
				$heightelem	= $('.img-responsive').height()/2;
				$heightthis = $heightwin - $heightelem;
				$(current.opts.$orig).focus();
				$('html,body').animate({
					//scrollTop: $(this.element).offset().top -150
					//scrollTop:$(this.element).offset().top - $(window).height() / 2
					scrollTop: $(current.opts.$orig).offset().top - $heightthis
				}, 1000);
			}

With v. 5 I've error on $(current.opts.$orig).focus();

Can you me show a link where I can try to resolve my code?

Thank you.

**Comment by fancyapps** [comment]
Hi,

Well, do you really need that? The focus is already placed on the corresponding trigger element after closing Fancybox. Just make sure you are using the latest version!

**Reply by ieraora** [reply]
I'm sorry, now I've Understand! I connfirm, focus is placed.
